bring-it-up/bring-it-up
# FE#25 Create Specialties Component

## Description
#### Summary:
Adding a specialties component that displays a service's areas of specialty as chips.

#### Changes:
- Adds Specialties.tsx in the components folder
- Adds the specialties component to TestService.tsx

#### How to test this PR:
1. Go to the services page
2. Select a service
3. The service's specialties should be visible

## Screenshots:
<img width="741" alt="Screen Shot 2023-03-19 at 9 14 47 PM" src="https://user-images.githubusercontent.com/98232527/226787685-001b01a2-420f-4a81-9b07-182d7fda0509.png">

<img width="741" alt="Screen Shot 2023-03-19 at 9 14 24 PM" src="https://user-images.githubusercontent.com/98232527/226787696-fe2002a0-e1d1-46e5-86c5-6438d34f6d4f.png">

## Checklist
